Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -469,8 +469,46 @@ namespace OpenInfraPlatform {
 			void convertIfcOpenCrossProfileDef(const EXPRESSReference<typename IfcEntityTypesT::IfcOpenCrossProfileDef>& profileDef,

```

Comment:
please wrap this function in conditional:


    // IfcOpenProfileDef exists starting IFC4x3
    #if defined(OIP_MODULE_EARLYBINDING_IFC4X1) || defined(OIP_MODULE_EARLYBINDING_IFC4) || defined(OIP_MODULE_EARLYBINDING_IFC2X3)
    #else
    /... doxycomment, function defintion, function body .../
    #endif

Reply:
Resolved: 0ffaa51